docs/painless/painless-guide/painless-ingest.asciidoc

```diff
@@ -96,3 +96,14 @@ supplied in the `value` parameter.
 ----
 String urlDecode(String value);
 ----
+
+===== URI decomposition
+Use the {ref}/uriparts-processor.html[URI parts processor] to decompose the URI string
```

Comment:
From looking at the existing documentation for the other processors, I don't see how these link references are resolved.

Reply:
```suggestion
Use the {ref}/uri-parts-processor.html[URI parts processor] to decompose the URI string
```

Reply:
@danhermann, looking at the syntax, it looks like you're missing a dash in the URL: https://www.elastic.co/guide/en/elasticsearch/reference/master/uri-parts-processor.html